src/App.jsx

```diff
@@ -1,5 +1,27 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import Home from './Components/Home';
+import Game from './Components/Game';
+import Collection from './Components/Collection';
+import Rules from './Components/Rules';
+import News from './Components/News';
+import NavBar from './Components/Navbar';
+import './reset.css';
+import './App.css';
+
 function App() {
-  return <h1>Hello there!</h1>;
+  return (
+    <>
+      <NavBar />
+      <Switch>
+        <Route path="/" exact component={Home} />
```

Comment:
Does it work like that (with the "exact" after the path )? because I know that we write it like this ( " exact path=' /' )